src/foundry/foundry.js/pixi/containers/baseGrid.d.ts

```diff
@@ -141,3 +104,53 @@ declare class BaseGrid extends PIXI.Container {
    */
   getNeighbors(row: number, col: number): [number, number][];
```

Comment:
I guess the return type could be `PointArray[]`? Also applies to the deriving classes which override this method.